src/index.js

```diff
@@ -14,6 +15,25 @@ module.exports = class WebappWebpackPlugin {
       favicons: {},
       prefix: 'assets/',
     }, options);
+
+    this.tags = {};
+    this.tags.promise = new Promise((resolve, reject) => {
+      this.tags.resolve = resolve;
+      this.tags.reject = reject;
+    });
+    this.tags.promise.catch(() => {});
```

Comment:
If i did it like this:
```js
this.tags.promise = new Promise((resolve, reject) => {
  this.tags.resolve = resolve;
  this.tags.reject = reject;
}).catch(() => {});
```
Then yea, rejection will be shallowed and rule loader will get `undefined` in `then`.

Here though, because original promise is exposed (and not `catch`ed), both `catch`es are called on a rejection. (Checked it before.)